src/ui-shell/header/header-navigation-items.interface.ts

```diff
@@ -0,0 +1,20 @@
+// Used for both HeaderMenus and HeaderItems in the header navigation to generate
+// items and menus through a model.
```

Comment:
This should be a full `/**` documentation block - that way it'll show up in the generated docs